vars/common-vars.yml

```diff
@@ -57,5 +57,5 @@ ucp_keystone_admin_password: 'password123'
 openstack_keystone_admin_password: 'password123'
 ucp_shipyard_keystone_password: 'password123'
 
-# Flag to run pod rally tests (default false)
-run_rally_tests: false
+# Flag to run or disable all helm tests (default true)
+run_tests: true
```

Comment:
This is not really an option we should expose in common vars. Common vars is not a graveyard for vars... It's the place were we put variables that are common to multiple roles.

Here I would say, with the fact it's not used by multiple roles (where is this variable used btw?) it should probably go into the role defaults.